skunk/rust/skunk/src/ir_gen/codegen/examples_codegen.rs

```diff
@@ -286,6 +290,11 @@ pub fn main_for_examples<'ctx>(context: &'ctx Context, target_machine: &TargetMa
       cg.builder.build_return(Some(&context.i32_type().const_int(-(1 as i32) as u64, true)));
       
       cg.builder.position_at_end(next);
+
+      let example_count_name = name.clone() + "__get_example_count";
+      let example_count_fn = submodule.get_function(&example_count_name).unwrap();
+      let count = cg.builder.build_call(example_count_fn, &[], "count").try_as_basic_value().left().unwrap().into_int_value();
```

Comment:
Aside: Can the unwraps here actually fail?

Reply:
They can! If they do, it means that the function doesn't have the signature it was called with.

So there's ideal behavior (a nicely formatted panic message describing where the problem occurred), unwrap behavior (which panics and gives the location but not a nicely formatted message), and Result behavior (poor localization, no information you don't manually add to the Result).

Probably we should switch to panic but for a lot of these a failure would be extremely surprising; I figure in those cases unwrap behavior is probably good enough?